udata/settings.py

```diff
@@ -108,6 +108,11 @@ class Defaults(object):
     # Flask WTF settings
     CSRF_SESSION_KEY = 'Default uData csrf key'
 
+    # Flask-Sitemap settings
+    # TODO: chose between explicit or automagic for params-less endpoints
+    # SITEMAP_INCLUDE_RULES_WITHOUT_PARAMS = False
```

Comment:
Why is this commented? If it's the default value, maybe we can uncomment it?

Reply:
Oh, I forgot, we ha a choice to make between:
- sticking with the current behavior: explicitely export urls without params to the sitemap
- sticking with the new default flask-sitemap behavior: urls without parameters are automatically exported to the sitemap. We only need to adds urls with parameters

Which one suits you ?

Reply:
No idea. Whatever takes less code ;-)

Reply:
OK, second one it is :+1: